### PR TITLE
[APP] visualping #6013

### PR DESCRIPTION
**Pull request by lcaresia**
### <samp>🤖 Generated by Copilot at e75cc84</samp>

This pull request updates the `visualping` component to use the new `.mjs` file extension for app and source modules. It also removes the unused `.gitignore` and `visualping.app.ts` files, and fixes the `package.json` file to point to the correct app module file.

### <samp>🤖 Generated by Copilot at e75cc84</samp>

> _`visualping.app`_
> _Exports an object for API_
> _Spring of new features_

### <samp>🤖 Generated by Copilot at e75cc84</samp>

*  Add app module for VisualPing API ([link](https://github.com/PipedreamHQ/pipedream/pull/6062/files?diff=unified&w=0#diff-da36e36292b91fee1b2e51d6661d1ed659a1d61f749ba17e54a28e9920cdb530R1-R77))
* Delete unused app module file ([link](https://github.com/PipedreamHQ/pipedream/pull/6062/files?diff=unified&w=0#diff-4306fa68cd251e45aecfcd3cb1ea3297614d1f60f35b4d97c6bc2cfe1dd842f7))
* Update `package.json` file to point to correct app module file ([link](https://github.com/PipedreamHQ/pipedream/pull/6062/files?diff=unified&w=0#diff-2ac3f1febef8b4610703afc65e9a3b35833f712d38bd260a030374f95947d83fL5-R9))
* Add source module for "New Job Created" event ([link](https://github.com/PipedreamHQ/pipedream/pull/6062/files?diff=unified&w=0#diff-6bcecab02c6f4598c79633588182ae3345fa8b0bb938a35a3b3414f3c205f96fR1-R74))
* Delete unnecessary `.gitignore` file ([link](https://github.com/PipedreamHQ/pipedream/pull/6062/files?diff=unified&w=0#diff-bb115108c531f089a50d59c84b3aecd6d2bf7d1733e931ca26b4621787ded9d5))

### CODE REVIEW

File: components/visualping/app/visualping.app.mjs [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import { axios } from "@pipedream/platform";
+
+export default {
+  type: "app",
+  app: "visualping",
+  propDefinitions: {
+    jobId: {
+      label: "Job ID",
+      description: "The job ID",
```
Comment (dylburger) [inline components/visualping/app/visualping.app.mjs:9]:
```suggestion
      description: "The job ID.",
```

Comment (dylburger) [reply components/visualping/app/visualping.app.mjs:9]: Added period to the end of the sentence (Google's Developer docs style guide)

File: components/visualping/sources/new-job-created/new-job-created.mjs [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import visualping from "../../app/visualping.app.mjs";
+import { DEFAULT_POLLING_SOURCE_TIMER_INTERVAL } from "@pipedream/platform";
+
+export default {
+  name: "New Job Created",
+  version: "0.0.1",
+  key: "visualping-new-job-created",
+  description: "Emit new event for each new job created.",
```
Comment (dylburger) [inline components/visualping/sources/new-job-created/new-job-created.mjs:8]:
```suggestion
  description: "Emit a new event for each new job created.",
```

Comment (dylburger) [reply components/visualping/sources/new-job-created/new-job-created.mjs:8]: Changed "Emit new event" to "Emit a new event" (Google's Developer docs style guide)

File: components/visualping/sources/new-job-event/new-job-created.mjs [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import visualping from "../../app/visualping.app.mjs";
+import { DEFAULT_POLLING_SOURCE_TIMER_INTERVAL } from "@pipedream/platform";
+
+export default {
+  name: "New Job Event",
+  version: "0.0.1",
+  key: "visualping-new-job-event",
+  description: "Emit new event for each new job event.",
```
Comment (dylburger) [inline components/visualping/sources/new-job-event/new-job-created.mjs:8]:
```suggestion
  description: "Emit a new event for each new job event.",
```

Comment (dylburger) [reply components/visualping/sources/new-job-event/new-job-created.mjs:8]: Changed "Emit new event" to "Emit a new event" (Google's Developer docs style guide)

File: components/visualping/package.json [hunk 4]
```diff
@@ -1,17 +1,16 @@
 {
   "name": "@pipedream/visualping",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "description": "Pipedream VisualPing Components",
-  "main": "dist/app/visualping.app.mjs",
+  "main": "app/visualping.app.mjs",
```
Comment (feyzullah) [inline components/visualping/package.json:5]:
```suggestion
  "main": "visualping.app.mjs",
```

File: components/visualping/sources/new-job-event/new-job-created.mjs [hunk 5]
```diff
@@ -0,0 +1,43 @@
+import visualping from "../../app/visualping.app.mjs";
+import { DEFAULT_POLLING_SOURCE_TIMER_INTERVAL } from "@pipedream/platform";
+
+export default {
+  name: "New Job Event",
+  version: "0.0.1",
+  key: "visualping-new-job-event",
```
Comment (feyzullah) [inline components/visualping/sources/new-job-event/new-job-created.mjs:7]: It seems file name should be changed to `new-job-event.mjs`

File: pnpm-lock.yaml [hunk 6]
```diff
@@ -1177,6 +1177,9 @@ importers:
     dependencies:
```
Comment (feyzullah) [inline pnpm-lock.yaml:1177]: It seems dependencies for visualping were not added to `pnpm-lock.yaml` file. Can you run `npx pnpm install`?

**Review by feyzullah — CHANGES_REQUESTED** [review]
Hey @lcaresia, I have added a few comments. Can you please check?

File: components/visualping/package.json [hunk 7]
```diff
@@ -1,17 +1,19 @@
 {
   "name": "@pipedream/visualping",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "description": "Pipedream VisualPing Components",
-  "main": "dist/app/visualping.app.mjs",
+  "main": "visualping.app.mjs",
```
Comment (feyzullah) [inline components/visualping/package.json:5]:
```suggestion
  "main": "app/visualping.app.mjs",
```

**Review by feyzullah — DISMISSED** [review]

**Comment by vunguyenhung** [comment]
Hello everyone, I have tested this PR and there're some test cases failed or needed improvement. 

Please check the test report below for more information
[Visualping_6013_1023.pdf](https://github.com/PipedreamHQ/pipedream/files/11279520/Visualping_6013_1023.pdf)

File: components/visualping/app/visualping.app.mjs [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+
+        return jobs.map((job) => ({
+          value: job.id,
+          label: job.url,
+        }));
+      },
+    },
+    workspaceId: {
+      label: "Workspace ID",
+      description: "The workspace ID",
```
Comment (dylburger) [inline components/visualping/app/visualping.app.mjs:30]:
```suggestion
      description: "The workspace ID.",
```

Comment (dylburger) [reply components/visualping/app/visualping.app.mjs:30]: Added a period at the end of the sentence.

File: components/visualping/app/visualping.app.mjs [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+        });
+
+        return jobs.map((job) => ({
+          value: job.id,
+          label: job.url,
+        }));
+      },
+    },
+    workspaceId: {
+      label: "Workspace ID",
```
Comment (dylburger) [inline components/visualping/app/visualping.app.mjs:29]:
```suggestion
      label: "Workspace ID.",
```

Comment (dylburger) [reply components/visualping/app/visualping.app.mjs:29]: Added a period at the end of the sentence.

**Comment by vunguyenhung** [comment]
Hi everyone, all test cases are passed! Ready for release! 

Test report
[Visualping_6013_1653.pdf](https://github.com/PipedreamHQ/pipedream/files/11321196/Visualping_6013_1653.pdf)

**Review by feyzullah — APPROVED** [review]

**Review by feyzullah — APPROVED** [review]

**Pull request merged**